lib/__tests__/build-files.tests.js

```diff
@@ -34,7 +34,7 @@ beforeAll(() => {
   generateSite();
   return Promise.all([
     glob(docsDir + '/**/*.md'),
-    glob(buildDir + '/' + siteConfig.projectName + '/docs/*.html'),
+    glob(buildDir + '/' + siteConfig.projectName + '/docs/**/*.html'),
```

Comment:
Why did you make this change here?

Reply:
Hi @JoelMarcey ! Thanks for your feedback :-)

This change has been made to include html files which are in subdirectories like `/docs/en/about-slash.html` and not just `/docs/about-slash.html`, to ensure that not only html files with redirect have a doctype.